### PR TITLE
Nav-branch

### PR DESCRIPTION
**Pull request by kayjeee**
Modified the bookstore application to have:

A Navigation bar.
Three content sections:
Books list.
Added book form.
Contact info.
Used this wireframe as a guide:
Made sure that when you click on any of the links, the main area changed to  display only one of the three content sections. (Hint: simply addad and removed CSS classes to display or hide the content.)
, didnt use any library or framework, use only plain JavaScript and CSS.

Student Success has given me permission to continue this week solo. If you have any questions please reach out to Marco: success@microverse.org
the screenshot can be found at
https://drive.google.com/file/d/13dUbJ-JogtcChz0Mm10ost5vkD2TB2wb/view?usp=sharing

cheers and Reviewing!👏👏👏

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome books</title>
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
+    <<nav class="nav-bar">
```
Comment (tugsem) [inline index.html]: - [x] Kindly remove the extra `<` at the start of this line as it is a typo and can be seen on the page clearly.

File: index.js [hunk 2]
```diff
@@ -1,24 +1,140 @@
...
+  if (section) {
+    section.style.display = 'flex';
+  }
+};
+
+navContact.addEventListener('click', () => {
+  const navItems = navBar.querySelectorAll('.nav-item');
+  const sections = document.querySelectorAll('.section');
+  handleNavClick(navContact, navItems, sections);
+});
```
Comment (tugsem) [inline index.js]: - [ ] When the contact link is clicked on the navigation bar, nothing happens. Kindly make sure you reached correct elements in the dom and return the right section.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+  display: flex;
+  justify-content: end;
+}
+
+footer {
+  width: 100%;
+  border: 2px solid black;
+  margin-top: 10px;
+  padding: 5px 10px;
+}
```
Comment (tugsem) [inline styles.css:193]: - [ ] Footer should be fixed at the bottom of the page as in the given wireframe. Kindly fix its position. 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @kayjeee  👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of github flow
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (kayjeee) [reply styles.css:193]: fix it thank you 👏
 You can find this project here:  GitHub: [@kayjeee](https://github.com/kayjeee/bookstore).  Please leave a star it helps. Thanks for checking out my code!

File: index.js (hunk 2 above)
Comment (kayjeee) [reply index.js]: fix it thank you 👏
 You can find this project here:  GitHub: [@kayjeee](https://github.com/kayjeee/bookstore).  Please leave a star it helps. Thanks for checking out my code!

File: index.html (hunk 1 above)
Comment (kayjeee) [reply index.html]: fix it thank you 👏
 You can find this project here: [https://github.com/kayjeee/bookstore].  Please leave a star it helps. Thanks for checking out my code!

**Comment by kayjeee** [comment]
Please check it again 👏


 You can find this project here:  GitHub: [@kayjeee](https://github.com/kayjeee/bookstore).  Please leave a star it helps. Thanks for checking out my code!

File: index.js [hunk 4]
```diff
@@ -1,24 +1,140 @@
...
+  contactSection.style.display = 'none';
+});
+navBooks.addEventListener('click', () => {
+  navHome.style.color = 'black';
+  navBooks.style.color = 'blue';
+  navContact.style.color = 'black';
+  homeSection.style.display = 'none';
+  bookSection.style.display = 'flex';
+  contactSection.style.display = 'none';
+});
```
Comment (AmaduKamara) [inline index.js:108]: - [ ] Just as the previous reviewer highlighted, when the contact link is clicked, the contact section is expected to show on the page but it's not at the moment and I found out that you forgot to add this code that shows the contact section just as you did for the other two sections:
- [x] Add these codes below line 108 so things will work fine as expected 👍🏽

```js
navContact.addEventListener("click", () => {
  navHome.style.color = "black";
  navBooks.style.color = "blue";
  navContact.style.color = "black";
  homeSection.style.display = "none";
  bookSection.style.display = "none";
  contactSection.style.display = "block";
});
``

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @kayjeee,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed 👍🏽
✅ The app is styled nicely 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.js (hunk 4 above)
Comment (kayjeee) [reply index.js:108]: Thank you for the kind words. Please check it again 👏


 You can find this project here:  GitHub: [@kayjeee](https://github.com/kayjeee/bookstore).  Please leave a star it helps. Thanks for checking out my code!

**Review by sumairq — APPROVED** [review]
### Hi @kayjeee , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kayjeee** [comment]
great work

**Pull request merged**